ChibiOS_3.0.5/os/common/ports/ARMCMx/compilers/GCC/rules.mk

```diff
@@ -312,6 +312,13 @@ clean:
 #
 # Include the dependency files, should be the last of the makefile
 #
--include $(shell mkdir .dep 2>/dev/null) $(wildcard .dep/*)

```

Comment:
I really don't enjoy changing ChibioOS core files, this just makes life harder when upgrading ChibiOS.